ext/mixed/js/display.js

```diff
@@ -18,7 +18,7 @@
 
 /*global docRangeFromPoint, docSentenceExtract
 apiKanjiFind, apiTermsFind, apiNoteView, apiOptionsGet, apiDefinitionsAddable, apiDefinitionAdd
-apiScreenshotGet, apiForward, apiAudioGetUrl
+apiScreenshotGet, apiForward, apiAudioGetUri
```

Comment:
Should these be made one per line, such as
```javascript
/* global
 * apiScreenshotGet
 * apiForward
 * apiAudioGetUri
 */ 
```

At the risk of taking up a bit more space, this would reduce the amount of conflicts to resolve.

Reply:
I would consider that outside of the scope of this change, but that sounds like a good idea. Particularly because, long-term, the number of global definitions should eventually be reduced. For example:

```js
/* global
 * apiScreenshotGet
 * apiForward
 * apiAudioGetUri
 */
```

Will likely eventually become:

```js
/* global
 * api
 */
```